### PR TITLE
Added new Event Vendor for Network Schema

### PR DESCRIPTION
**Pull request by devikamehra**
Required items, please complete
   
   Change(s):
   - Added new FORCEPOINT Event Vendor for Network Schema

   Reason for Change(s):
   - Adding new ASIM Parser

   Version Updated:
   - NA

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

File: ASIM/dev/ASimTester/ASimTester.csv [hunk 1]
```diff
@@ -667,7 +667,7 @@ EventStartTime,datetime,Mandatory,NetworkSession,,,
 EventSubType,string,Optional,NetworkSession,Enumerated,Start|End|,
 EventType,string,Mandatory,NetworkSession,Enumerated,NetworkSession|L2NetworkSession|EndpointNetworkSession|Flow,
 EventUid,string,Recommended,NetworkSession,,,
-EventVendor,string,Mandatory,NetworkSession,Enumerated,Fortinet|AppGate|Palo Alto|Microsoft|Zscaler|AWS|Vectra AI|WatchGuard|Cisco|Corelight|Check Point,
+EventVendor,string,Mandatory,NetworkSession,Enumerated,Fortinet|AppGate|Palo Alto|Microsoft|Zscaler|AWS|Vectra AI|WatchGuard|Cisco|Corelight|Check Point|FORCEPOINT,
```
Comment (oshezaf) [inline ASIM/dev/ASimTester/ASimTester.csv:670]: Why all capital?

**Review by oshezaf — CHANGES_REQUESTED** [review]

File: ASIM/dev/ASimTester/ASimTester.csv (hunk 1 above)
Comment (devikamehra) [reply ASIM/dev/ASimTester/ASimTester.csv:670]: The source is sending it in this format.

Comment (oshezaf) [reply ASIM/dev/ASimTester/ASimTester.csv:670]: Sure. This is why I asked you to use a static value in the parser as well... we are here to fix source mischievous behavior :-)

Comment (devikamehra) [reply ASIM/dev/ASimTester/ASimTester.csv:670]: Understood :-)
I have updated in the Parser as well.

Comment (oshezaf) [reply ASIM/dev/ASimTester/ASimTester.csv:670]: Did you push for this PR? I don't see the change here yet.

Comment (devikamehra) [reply ASIM/dev/ASimTester/ASimTester.csv]: I have pushed the changes just now. Also, added ForcepointId as well in DvcIdType as it part of the similar effort.

**Review by oshezaf — APPROVED** [review]

**Pull request merged**